scripts/generate-types.js

```diff
@@ -62,6 +62,8 @@ webhooks.forEach(({ name, actions, examples }) => {
 conditionalType.push("never");
 
 const getWebhookPayloadTypeFromEvent = `
+${doNotEditThisFileDisclaimer}
```

Comment:
I think `doNotEditThisFileDisclaimer` needs to be defined in this file for it to work?

Reply:
> I think `doNotEditThisFileDisclaimer` needs to be defined in this file for it to work?

It was already defined. Only missing on some generated files:

https://github.com/octokit/webhooks.js/blob/96c33db3e5a47d0a221afe7d8eca32c6dd58144a/scripts/generate-types.js#L9-L11